### PR TITLE
KIALI-980 :First of our low risk library updates including: cytoscape

### PR DESCRIPTION
**Pull request by mtho11**
This is the first of several PRS to update the various libraries This one includes: cytoscape and the cytoscape plugins. Also url-search-params and visibilityjs. And lastly, remove the unused cytoscape-panzoom.

** Issue reference **

https://issues.jboss.org/browse/KIALI-980

** Backwards compatible? **

Yes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,16 +8,16 @@
...
     "csx": "9.0.0",
-    "cytoscape": "3.2.14",
+    "cytoscape": "3.2.16",
     "cytoscape-canvas": "3.0.1",
     "cytoscape-cola": "2.2.3",
     "cytoscape-cose-bilkent": "4.0.0",
     "cytoscape-dagre": "2.2.1",
-    "cytoscape-panzoom": "2.5.2",
-    "cytoscape-popper": "1.0.1",
+    "cytoscape-panzoom": "2.5.3",
```
Comment (josejulio) [inline package.json]: We don't use this anymore, forgot to remove when removing that widget. Could you do that now or should i do that in a separate PR?

File: package.json [hunk 2]
```diff
@@ -8,16 +8,16 @@
...
+    "cytoscape-popper": "1.0.2",
```
Comment (josejulio) [inline package.json]: I think we don't use this anymore. IIRC @jshaughn removed it.

File: package.json (hunk 1 above)
Comment (josejulio) [reply package.json]: Stopped using that since https://github.com/kiali/kiali-ui/pull/631

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#44](https://paste.fedoraproject.org/paste/S4QqURxRPnhAvbjVC0f4AA)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1252]](https://paste.fedoraproject.org/paste/fYJx0y7hjaWpVeqSeSTSjQ)
  * :heavy_check_mark: Kiali-on-Openshift [#[596]](https://paste.fedoraproject.org/paste/KfZcLjSv4yTpEHUH7UWtCA)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1023]](https://paste.fedoraproject.org/paste/e545nSeH4gT7HR5ICKAIBA)
  * :red_circle: run-kiali-e2e-tests [#[69]](https://paste.fedoraproject.org/paste/ByBiVDwnagZGFkGcphFMEQ)


KIALI_NAME=kialiuipr669-istio-system

File: package.json (hunk 2 above)
Comment (josejulio) [reply package.json]: It was removed in https://github.com/kiali/kiali-ui/pull/650

File: package.json (hunk 1 above)
Comment (mtho11) [reply package.json]: sure. I'll just remove it.

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#46](https://paste.fedoraproject.org/paste/UfiHD5ETbYQE8f2Bv60Iuw)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1254]](https://paste.fedoraproject.org/paste/jDScmhKhfxPCLDoWynsWdQ)
  * :heavy_check_mark: Kiali-on-Openshift [#[598]](https://paste.fedoraproject.org/paste/KdGdfN4ji~LeA0mu89qTWQ)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1025]](https://paste.fedoraproject.org/paste/kHZH4mo3G-S1qjE2lj7GOQ)
  * :red_circle: run-kiali-e2e-tests [#[71]](https://paste.fedoraproject.org/paste/fJ21fTP9wtYQAVKuAc1fpA)


KIALI_NAME=kialiuipr669-istio-system

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#47](https://paste.fedoraproject.org/paste/BZiEjgaj9n5nGpR6Apw3hw)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1255]](https://paste.fedoraproject.org/paste/R~hpgi59yN9EbcJ68ssFNA)
  * :heavy_check_mark: Kiali-on-Openshift [#[599]](https://paste.fedoraproject.org/paste/-pOREIYF0mRsFhVANVYwRg)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1026]](https://paste.fedoraproject.org/paste/G-GMzz9drjVILYg6Cq1kWw)
  * :red_circle: run-kiali-e2e-tests [#[72]](https://paste.fedoraproject.org/paste/AcCwBUjtKBDzytUzyvbAxQ)


KIALI_NAME=kialiuipr669-istio-system
